src/components/forms/lock_actions/LockForm/components/LockPreviewModal/LockPreviewModal.vue

```diff
@@ -73,6 +84,8 @@ function handleClose() {
 
 function handleSuccess() {
   lockConfirmed.value = true;
+  refetchLockInfo.value();
+  resetState();
```

Comment:
The `resetState` function resets the amounts (it looks way better when amounts are zero after a successful lock)